### PR TITLE
Adapt API consumer for mobile CMS & implement Gallery models

### PR DESCRIPTION
**Pull request by zachgarwood**
I originally tried to separate it out into two PRs, but the code is kind of intertwined, so this covers both [MA-90](https://art-institute-of-chicago.atlassian.net/browse/MA-90) & [MA-93](https://art-institute-of-chicago.atlassian.net/browse/MA-93).

[MA-90]: https://art-institute-of-chicago.atlassian.net/browse/MA-90?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[MA-93]: https://art-institute-of-chicago.atlassian.net/browse/MA-93?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/Libraries/Api/Models/BaseApiModel.php [hunk 1]
```diff
@@ -113,27 +118,70 @@ public function __construct(array $attributes = [])
...
     {
-        //
+        return $query->where("{$this->getTable()}.published", false);
     }
 
     public function scopeDraft(Builder $query): Builder
     {
-        //
+        return $query->onlyTrashed();
     }
```
Comment (nikhiltri) [inline app/Libraries/Api/Models/BaseApiModel.php]: The bodies of these methods should be swapped, yea?

File: app/Libraries/SmartyPants.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Libraries;
+
+use Michelf\SmartyPants as BasePants;
+
+class SmartyPants extends BasePants
+{
+    /**
+     * Override parent method to prevent it from transforming `null` to empty string.
```
Comment (nikhiltri) [inline app/Libraries/SmartyPants.php:10]: 🎉

**Review by nikhiltri — CHANGES_REQUESTED** [review]
Nice work! 👏🏾 ⭐

File: app/Libraries/Api/Models/BaseApiModel.php (hunk 1 above)
Comment (zachgarwood) [reply app/Libraries/Api/Models/BaseApiModel.php]: Yup! Nice catch.

**Review by nikhiltri — APPROVED** [review]
Looks good! ⭐

**Pull request merged**